packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/NodeTreeFilter/index.js

```diff
@@ -40,41 +40,41 @@ export default class NodeTreeFilter extends PureComponent {
             .filter(presetName => (presetName !== 'default'))
             .map(presetName => ({
                     value: presets[presetName].baseNodeType,
-                    label: presets[presetName].ui && presets[presetName].ui.label || '[' + presetName + ']',
-                    icon: presets[presetName].ui && presets[presetName].ui.icon || null,
+                    label: $get('ui.label', presets[presetName]) || '[' + presetName + ']',
```

Comment:
I tried to provide an example, I meant this: `$get([presetName, 'ui', 'label'], presets)`

Reply:
Ah right. But is that any better than `$get('ui.label', presets[presetName])`? Anyways, I'll change it